### PR TITLE
Repository Archived text title center align

### PR DESCRIPTION
**Pull request by puni9869**
Archive text title center align

<details>
  <summary>Screen shots</summary>
  
 Before
image

After 
image

image

BTW On github
image

</details>

### CODE REVIEW

**Review by denyskon — APPROVED** [review]

**Review by yardenshoham — APPROVED** [review]

File: templates/repo/diff/compare.tmpl [hunk 1]
```diff
@@ -214,7 +214,7 @@
 					<button class="ui button green show-form">{{.locale.Tr "repo.pulls.new"}}</button>
 				</div>
 			{{else if .Repository.IsArchived}}
-				<div class="ui warning message">
+				<div class="ui warning message gt-tc">
```
Comment (delvh) [inline templates/repo/diff/compare.tmpl]: Erm… Please wait for #25763.

**Review by delvh — DISMISSED** [review]

File: templates/repo/diff/compare.tmpl (hunk 1 above)
Comment (puni9869) [reply templates/repo/diff/compare.tmpl]: Is the css rule is going to be changed? `gt-tc`

Comment (puni9869) [reply templates/repo/diff/compare.tmpl]: Oh I see rules are changing

Comment (puni9869) [reply templates/repo/diff/compare.tmpl]: I have updated the code according to `gt-text-center`

**Review by silverwind — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Pull request merged**